app/scrapers/stackoverflow.py

```diff
@@ -17,7 +18,7 @@ def parseResponse(self, soup):
         urls = []
         for div in soup.findAll('div', {'class': 'result-link'}):
             a = div.find('a')
-            link = self.result_url + str(a.get('href'))
+            link = 'https://stackoverflow.com' + str(a.get('href'))
```

Comment:
I think this is not the best way to do, cause some links might be external and some might already contain http:/...